### ISSUE TITLE
Potential bug in getAverageReviewInterval

### ISSUE THREAD

**Issue by wontonst** [monitise-mea/gerrit-dashboard-server#31]
Looking at ```function getAverageReviewInterval (callback)```

This logic doesn't look right to me

                        reviewCreateTime = moment(mergedReviewsList[review].created);
                        reviewUpdateTime = moment(mergedReviewsList[review].updated);

                        interval = reviewUpdateTime.diff(reviewCreateTime, TIME_PERIOD_TYPE);

A commit that has been merged can still be commented on, and adding a comment changes the last updated timestamp. This shows the time between create and last updated but what we want is time between create and merge.

**Comment by wontonst** [comment]
Yeah I'm convinced this logic is wrong. I've opened a stackoverflow dealing with this problem: http://stackoverflow.com/questions/41642244/how-to-get-review-time-information-from-gerrit/41642647#41642647

**Comment by tunix** [comment]
Hi @wontonst,

Thanks for sharing this with us! We'll look into it shortly. Are you willing to send a PR btw?

**Comment by wontonst** [comment]
I can put it on my todo but I don't have the bandwidth at the moment. I have a working solution but it's in python.
